R/create_SE.R

```diff
@@ -120,25 +119,30 @@ create_SE <- function(df_,
   ## The mapping_entries contain all exhaustive combinations of treatments 
   ## and cells. Not all conditions will actually exist in the data, so filter 
   ## out those that do not exist. 
-  treated <- treated[rownames(treated) %in% unique(groupings), ]
+  treated_rows <- which(treated$rownames %in% unique(groupings))
```

Comment:
Using:
`treated <- treated[rownames %in% unique(groupings)]` without a comma at the end should work for data.table